tests/component/test_device_properties.py

```diff
@@ -6,92 +6,92 @@
 from nidaqmx.system import Device
 
 
-def test___constructed_device___get_property___returns_value():
-    device = Device("bridgeTester")
+def test___constructed_device___get_property___returns_value(generate_device):
+    device = generate_device("bridgeTester")
 
     assert device.product_type == "PXIe-4331"
 
 
-def test___nonexistent_device___get_property___raises_invalid_device_id():
-    device = Device("NonexistentDevice")
+def test___nonexistent_device___get_property___raises_invalid_device_id(generate_device):
+    device = generate_device("NonexistentDevice")
```

Comment:
Is this why you removed the skip code from all the fixtures? To avoid skipping tests that deliberately construct objects for non-existent devices/tasks/channels/scales?

I don't think `generate_device` is adding much value. It doesn't do any cleanup, and in my opinion constructing `Device` directly with `**init_kwargs` would be clearer. Likewise for `PersistedChannel`, `PersistedScale`, etc.

Reply:
I have update the fixture as per your suggestion.